deploy/utils.sh

```diff
@@ -477,3 +478,35 @@ get_logs() {
     get_conjur_logs_container
     get_app_logs_container
 }
+
+# Return the pods information of the test-env without the headers
+# For example: 'test-app-5-fab52b20-0 secret-provider-0 1/1 Running 1 20m'
+get_pods_info() {
+  $cli_with_timeout get pods \
+    --namespace="$APP_NAMESPACE_NAME" \
+    --selector app=test-env \
+    --no-headers
+}
+
+# Return the pod name of a given namespace and app name
+# For example: 'secret-provider-0'
+get_pod_name() {
+  local namespace=$1
+  local selector=$2
+  local cli=${3-$cli_with_timeout}
```

Comment:
Use:
```sh
  local cli=${3:-$cli_with_timeout}
```
What you have works but isn't documented in man bash.  `:-` is the documented syntax.

Reply:
oh right, missed that. I usually remember this by knowing that the syntax is a smily face when the default value is empty (`:-}`)